src/ai/backend/manager/api/admin.py

```diff
@@ -170,5 +176,5 @@ def create_app(
     # (graphene transforms our object model into a textual representation)
     # This is useful for writing documentation!
     schema = graphene.Schema(query=Queries, mutation=Mutations, auto_camelcase=False)
```

Comment:
Main characteristic of this CLI invocation (which dumps GQL schema as a plaintext file) duplicates with `backend.ai mgr gql show` command, so I think it is better to just drop this whole execution block and encourage users to use `gql show` command instead.

Reply:
Applied!